src/concrete_solve.jl

```diff
@@ -690,6 +690,7 @@ function DiffEqBase._concrete_solve_adjoint(prob,alg,
                                  u0,p,args...;save_start=true,save_end=true,
                                  saveat = eltype(prob.tspan)[],
                                  save_idxs = nothing,
+                                 g = nothing,
                                  t0skip=zero(eltype(prob.tspan)), t1skip=zero(eltype(prob.tspan)),

```

Comment:
I'm not sure what these kwargs are? Should these be in the algorithm? I think the kwarg checker found a real issue 😅

Reply:
I am a bit confused -- I tried to fix the kwargs warning 

> Warning: Unrecognized keyword arguments found 

in 
https://buildkite.com/julialang/diffeqsensitivity-dot-jl/builds/743#3af4314a-160c-4c3d-adae-7d99ec5da380.
I thought that this warning got triggered because `g` (the instantaneous objective function) was a kwarg and not recognized by `solve`.

Do you have an idea where the error in the Core1 test group comes from? 
```julia

du06,dp6 = Zygote.gradient((u0,p)->sum(solve(prob,Tsit5(),u0=u0,p=p,abstol=1e-14,reltol=1e-14,saveat=0.1,sensealg=ForwardSensitivity())),u0,p)

@test du06 isa ChainRulesCore.NotImplemented
```
Evaluates to `nothing isa ChainRulesCore.NotImplemented` and fails. Seems to be independent of the shadowing methods and the changes in this PR?

Reply:
> I thought that this warning got triggered because g (the instantaneous objective function) was a kwarg and not recognized by solve.

Yes, it's not. The list is: https://github.com/SciML/DiffEqBase.jl/blob/master/src/solve.jl#L13-L67 `t0skip` is also not in the list. I'm not sure what they do either 😅

Reply:
Oh, is the warning triggered because the original solution call contains a "g=g"? Then I think I should add `g`
`t0skip`, and `t1skip` as options to `LSS()`. The times `t0skip` and `t1skip` are needed for the dilation term  (to reduce side effects)  as alternative options for windowing
https://github.com/SciML/DiffEqSensitivity.jl/blob/4151ba26d5032d7698fe4d5224ff11f3f2677d41/src/lss.jl#L308

Reply:
Yes exactly. It's trying to solidify the common args, with non-common args being pieces documented and held by specific algorithms.